manifests/init.pp

```diff
@@ -77,7 +77,7 @@
     } else {
       $real_conf_template = "${module_name}/winlogbeat1.yml.erb"
     }
-  } elsif $real_version == '5' {
+  } elsif $real_version == '5' or $real_version == '6' or $real_version == '7' {
```

Comment:
This logic is pretty complex. You should have a spec test that shows the right content is being used for a given version.

Reply:
Respectfully I don't feel this logic I've modified is particularly complex at all. There is complex logic before it, but this line is fairly straight forward.

My main issue is that I do not have a working Ruby environment to write/test specs with, so I won't be able to contribute a test here regardless.

Reply:
You can make this more readable by using `in` https://puppet.com/docs/puppet/6.10/lang_expressions.html#in:
```suggestion
  } elsif $real_version in ['5', '6', '7'] {
```

as an alternative, the version could be an int and we just use `> 4`, but that would require more changes on the code, so I'm fine with `in`.